gnupg/gnupg.py

```diff
@@ -496,6 +496,38 @@ def list_packets(self, raw_data):
                         result)
         return result
 
+    def sign_key(self, keyid, passphrase=None):
+        """ sign (an imported) public key - keyid, with default secret key
```

Comment:
It will be handy to have a `default_key` param like in the `sign` method, to be able to select the secret key to be used to sign with. I will be ok with just adding a comment with a TODO if this requires some work.